lib/y2lan_restart_common.pm

```diff
@@ -48,6 +48,7 @@ sub open_network_settings {
     type_string "yast2 lan\n";
     accept_warning_network_manager_default;
     assert_screen 'yast2_lan', 100;       # yast2 lan overview tab
+    send_key 'home';                      # select first device
```

Comment:
Still, I'm not sure if it guarantees selecting correct device, except the identified case. But I would really make it work, so we don't have to touch it if ever enable this test on bare-metal machine.

Reply:
Without seeing other scenarios it is hard to say, this is based on the assumption that the first would be the right one, any other strategy like assert_and_click  would require create needle for each product to click on the corresponding interface (if scroll is not needed). We can always deal with more specific bare-metal scenarios in future, I wanted to cover only ppc here (which already has a lot of problem and it is time-consuming to get correct verification).

Reply:
Alright, let's leave it for us in the future =)